docs/usage_guide/advanced_usage/modin_logging.rst

```diff
@@ -53,3 +53,28 @@ Disable Modin logging like so:
   LogMode.disable()
 
   # User code goes here
+
+Debugging from user defined functions:
+
+
+.. warning:: 
+  If developer attempts to use modin logging in the remote function/UDF for logging lower-level operators as below, multiple log directories(.modin/logs/job_**) would be created for each worker executing the remote function.
+.. code-block:: python
+
+  def Udf(df, partition_shape, *dfs):
+      from modin.config import LogMode, LogMemoryInterval, LogFileSize
+      LogMode.enable()
+      # User code goes here
+      return
```

Comment:
Let's declare a udf function and pass it in map or apply method like the user interacts with pandas API. The udf will be executed in a remote function.

```python
def udf(x):
    logger stuff
    return x + 1

modin_df.map(udf)
```

Reply:
Done.